.packit.yaml

```diff
@@ -18,7 +18,7 @@ srpm_build_deps:
 actions:
   create-archive:
   - bash -c "sed -i -r \"s/Source0:.+/Source0:\ fido-device-onboard-rs-${PACKIT_PROJECT_VERSION}.tar/\" fido-device-onboard.spec"
-  - bash -c "sed -i \"/Source1/d\" fido-device-onboard.spec"
+  - bash -c "sed -i \"/Source1/d ; /^# See make-vendored-tarfile.sh in upstream repo/d ;\" fido-device-onboard.spec"
```

Comment:
The comment in the spec is for users who may not be aware of where it comes from, I am not sure why we need to modify it as part of this process though.

Reply:
The original code removes the **Source1** line from the spec which I guess is not needed because it looks like we only build fedora packages in packit (so we don't need vendoring). This change also removes the comment just above **Source1** and has no effect in how the package is currently built, it just removes a comment related to **Source1** which has no sense to maintain as **Source1** is already deleted.  I realized about this when I was checking the generated spec by **make rpm** for the different distributions

Reply:
Because of eln we have vendored source in Fedora too:
https://src.fedoraproject.org/rpms/fido-device-onboard/c/4dcb8949985982348b9c967106ba484caa73cc4b?branch=rawhide

See PRs like this around eln:
https://src.fedoraproject.org/rpms/fido-device-onboard/pull-request/4